test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
- [ ] Please Delete this file #`test.md`#  here because we already have it inside the `.gitignore` file, as was stated in the project requirements.

Reply:
Thanks for the feedback, but I do not understand the statement **"Every comment with the [OPTIONAL] prefix is not crucial"**, can you please elaborate with an example.